test/support/integration_test.rb

```diff
@@ -16,6 +16,10 @@ def initialize(args)
   end
 
   def setup
```

Comment:
I've just noticed that there are a few integration tests that don't call `super` in their `setup` methods, e.g. `ElasticsearchMigrationTest`. Should we fix those now?